docker/README.md

```diff
@@ -0,0 +1,149 @@
+# Building KubeStellar deployment as a container

```

Comment:
Somewhere right up front there should be a big picture explanation of that this image is for.  I think that is running the kcp server and the kubestellar controllers.

Reply:
Here, we discuss the creation and use of a **KubeStellar** container image for running the **kcp** server and the **KubeStellar** controllers, and delivering the **kcp** and **KubeStellar** executables, and kcp client kube config to the host. This provides an alternative to using **KubeStellar** bootstrap script. Additionally, we are planning to use this images in the future to run **KubeStellar** as a service in a cluster.

Reply:
the text is indeed in the README on line 13.  I think that is satisfactory.  Let's move on for now.